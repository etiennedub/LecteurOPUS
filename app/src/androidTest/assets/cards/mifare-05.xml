<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04896065</List><List index="1">2A444980</List><List index="2">A748E000</List><List index="3">0000000F</List><List index="4">00000000</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">173096C9</List><List index="8">09C677F0</List><List index="9">80E0351F</List><List index="10">40000000</List><List index="11">22284F7E</List><List index="12">01C677F0</List><List index="13">80E0351F</List><List index="14">40000000</List><List index="15">2228651D</List></Pages></Mifare>