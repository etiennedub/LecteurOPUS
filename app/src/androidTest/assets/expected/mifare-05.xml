<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="28/11/2016" Time="17:11" Transfert="true" OperatorId="20" BusId="3" OperatorName="RTC" BusName="LEBUS 4" /><List index="1" Date="28/11/2016" Time="17:11" Transfert="true" OperatorId="20" BusId="3" OperatorName="RTC" BusName="LEBUS 4" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>